src/components/header/Header.jsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import './Header.css';
+
+function Header() {
+  return (
+    <div className="sm:px-25 md:px-60 lg:px-60 header text-center h-screen place-items-center">
```

Comment:
- The background image needs an overlay (similar to how it looks in the Figma file) so the words are easy to read. You could had to overlay to the <div> named "header"
- I couldn’t tell if the place-items-center class you put on this div is actually doing anything. When I removed it nothing changed. You already placed “text-center” on this div so I don’t think you need to any the place-items-center for the headers.